DuckDuckGo/HomePage/Model/HomePageSettings/UserBackgroundImagesManager.swift

```diff
@@ -110,7 +110,14 @@ final class UserBackgroundImagesManager: UserBackgroundImagesManaging {
         }
     }
 
+    private var imagesCache: [UserBackgroundImage: NSImage] = [:]
+    private var thumbnailsCache: [UserBackgroundImage: NSImage] = [:]
```

Comment:
These caches will hold up to 4 entries each

Reply:
As that limit is not enforced, should we document this with a comment? Or does the limit matter?